schema.yaml

```diff
@@ -14,7 +14,7 @@ x-google-marketplace:
   #
   # Important: it must have the same value as the version in the
   # Application manifest.
```

Comment:
We can now safely remove the above comment since [application.yaml](https://github.com/jetstack/jetstack-secure-gcm/blob/main/chart/jetstack-secure-gcm/templates/application.yaml)'s `version` field is now templated:

```yaml
kind: Application
spec:
  descriptor:
    version: "{{ index .Values "cert-manager" "image" "tag" }}"
```